src/substrait/gen/proto/algebra_pb2.py

```diff
@@ -1,350 +1,348 @@
 """Generated protocol buffer code."""
 from google.protobuf import descriptor as _descriptor
 from google.protobuf import descriptor_pool as _descriptor_pool
-from google.protobuf import runtime_version as _runtime_version
 from google.protobuf import symbol_database as _symbol_database
 from google.protobuf.internal import builder as _builder
-_runtime_version.ValidateProtobufRuntimeVersion(_runtime_version.Domain.PUBLIC, 5, 29, 5, '', 'proto/algebra.proto')
```

Comment:
interesting that this line gets removed. when I generate the protos by running `uv run ./gen_proto.sh` using the latest devcontainer from `main` branch I do still get this runtime validation line